### PR TITLE
Fix function definition for Waypoints and silence unneeded developer …

### PR DESCRIPTION
**Pull request by sbeeds**
…print

seems like I had a typo while merging the microai revamp.

### CODE REVIEW

File: source/r_alias.c [hunk 1]
```diff
@@ -533,7 +533,7 @@ void R_SetupAliasFrame (aliashdr_t *paliashdr, int frame, lerpdata_t *lerpdata)
 
 	if ((frame >= paliashdr->numframes) || (frame < 0))
 	{
-		Con_DPrintf ("R_AliasSetupFrame: no such frame %d for '%s'\n", frame, e->model->name);
```
Comment (MotoLegacy) [inline source/r_alias.c:536]: Why was this removed? What was being complained about? It should probably be properly fixed

**Review by MotoLegacy — CHANGES_REQUESTED** [review]

File: source/r_alias.c (hunk 1 above)
Comment (sbeeds) [reply source/r_alias.c:536]: `"R_AliasSetupFrame: no such frame 9 for 'models/props/rebar.mdl'"`
directly after this print the frame is set to 0.

Comment (sbeeds) [reply source/r_alias.c:536]: also, `"R_AliasSetupFrame: no such frame 2 for 'models/props/teddy.mdl"` on Kino/Warehouse2 and others

Comment (MotoLegacy) [reply source/r_alias.c:536]: Thanks. We should be actually fixing these, not just suppressing the warning. Can you redo the commit without this change and open separate issues to report these errors ?

Rebar is going to be nacht specific, teddy is because of a regression made by @blubs  that broke the sitting frame
